docs/src/pages/components/data-grid/editing/BasicEditingGrid.js

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+import {
+  randomCreatedDate,
+  randomTraderName,
+  randomUpdatedDate,
+} from '@material-ui/x-grid-data-generator';
+
+const columns = [
```

Comment:
How about we move the columns and rows at the bottom of the file so that when the developer opens a demo, he immediately sees the logic, and only after the data?

x the other cases.

Reply:
We would have eslint complaining that we use the variable before its declaration.